libraries/AP_Follow/AP_Follow.cpp

```diff
@@ -765,6 +749,27 @@ bool AP_Follow::handle_follow_target_message(const mavlink_message_t &msg)
         _target_heading_rate_degs = 0.0f;
     }
 
+    _target_pos_ned_m = target_pos_ned_cm * 0.01; // convert from cm to meters
+    _target_pos_ned_m.z = -packet.alt + origin.alt * 0.01;
```

Comment:
This really needs to deal with different frame types the way that handle_global_position_int does at line 624. I guess I missed this. It might make sense to extract the lines from handle_global_position_int() and make a "decode altitude" method (or something like that), I think the code should be the same.